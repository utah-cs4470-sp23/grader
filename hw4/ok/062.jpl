assert array[x : 1] f[x], ""
