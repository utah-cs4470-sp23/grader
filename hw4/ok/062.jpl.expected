(AssertCmd
 (ArrayLoopExpr x (IntExpr 1) (ArrayIndexExpr (VarExpr f) (VarExpr x)))
 "")
